src/status_im/init/core.cljs

```diff
@@ -186,6 +186,7 @@
                 pairing/installations status-module-initialized? device-UUID semaphores]
          :node/keys [status]
          :or   {network (get app-db :network)}} db
+        stored-pns      (:push-notifications/stored db)
```

Comment:
We need to save `:push-notifications/stored` when we sign in, otherwise when `process-stored-events` is called, it will be too late. Or we need to make things work with navigation when account is not yet signed in (maybe store an initial navigation command to be executed when the user signs in to the account). Right now I get:
```
12-04 17:06:47.707 15095 15386 D ReactNativeJS: DEBUG [status-im.notifications.core:144] - #6772 - handle-push-notification from: 0x04325367620ae20dd878dbb39f69f02c567d789dd21af8a88623dc5b529827c2812571c380a2cd8236a2851b8843d6486481166c39debf60a5d30b9099c66213e4 to: 0x043ca1c37213796da2cd9662798194fcceaa57f90010ff3fd0ce6550f6b0a024267b5b2576f7e84d76a4c38f1f40e73c2cbb2f5be889b9d76a210a519801be9880 current-public-key: 0x043ca1c37213796da2cd9662798194fcceaa57f90010ff3fd0ce6550f6b0a024267b5b2576f7e84d76a4c38f1f40e73c2cbb2f5be889b9d76a210a519801be9880
12-04 17:06:47.717 15095 15386 E ReactNativeJS: ERROR [status-im.utils.fx:30] - Merging fx with common-keys:  #{:db :status-im.ui.screens.navigation/navigate-to}
12-04 17:06:47.777 15095 15386 E ReactNativeJS: 're-frame: no handler registered for effect:', { ns: null,
12-04 17:06:47.777 15095 15386 E ReactNativeJS:   name: 'merging-fx-with-common-keys',
12-04 17:06:47.777 15095 15386 E ReactNativeJS:   fqn: 'merging-fx-with-common-keys',
12-04 17:06:47.777 15095 15386 E ReactNativeJS:   _hash: -1375695184,
12-04 17:06:47.777 15095 15386 E ReactNativeJS:   'cljs$lang$protocol_mask$partition0$': 2153775105,
12-04 17:06:47.777 15095 15386 E ReactNativeJS:   'cljs$lang$protocol_mask$partition1$': 4096 }, '. Ignoring.'
```